docs: Update quickstarts (#705)
kubermatic/kubeone: docs/quickstart-aws.md

Finally, we're going to install Kubernetes by using the `install` command and providing the configuration file and the Terraform output:

```bash
kubeone install config.yaml --tfjson <DIR-WITH-tfstate-FILE>
```

> **Note:** `--tfjson` accepts a file as well as a directory containing the

Reviewer: Should we mention the `.` trick here?

Author: Fixed. :slightly_smiling_face:

```
The machinedeployments "pool1" is invalid: metadata.resourceVersion: Invalid value: 0x0: must be specified for an update
```

> For a workaround, please follow the steps described in the [issue 593][scale_issue]. But the above will work with `kubectl` 1.16.x.

Reviewer: I'd combine this into one sentence such as:
```suggestion
For a workaround, please follow the steps described in the [issue 593][scale_issue] or upgrade to `kubectl` 1.16 or newer.
```

Author: Fixed. :slightly_smiling_face:
